eclide/ChildAttributeFrame.cpp

```diff
@@ -102,6 +102,12 @@ class CAttributeFrame : public CChildFrame, public boost::signals::trackable,
         m_dlgview.SetSource(ecl);
     }
 
+    void SetSel(int row, int col, int len)
```

Comment:
Added SetSel in two places to make it cleaner. Again I couldn't find how to put this in SourceView and call it correctly from where it was needed. Suggestions welcome!